src/server/initiate/controller.js

```diff
@@ -50,7 +41,7 @@ const initiateController = {
     return h
       .response({
         statusUrl: `${appBaseUrl}/status/${uploadId}`,
-        uploadUrl: `${appBaseUrl}/upload/${uploadId}`,
+        uploadAndScanUrl: `${appBaseUrl}/upload-and-scan/${uploadId}`,
```

Comment:
Do we need this rename? The path rename we need but why the field?

Reply:
I guess there is this whole rename of the upload so I guess that is fine

Reply:
yeah more comms around what this does, especially if you just use the variable `initDetail.uploadAndScan`